pyrasterframes/src/main/python/docs/ipython.pymd

```diff
@@ -0,0 +1,70 @@
+# IPython/Jupyter Extensions
+
+The `pyrasterframes.rf_ipython` module injects a number of visualization extensions into the IPython environment, enhancing visualization of `DataFrame`s and `Tile`s.
+
+By default, the last expression's result in a IPython cell is passed to the `IPython.display.display` function. This function in turn looks for a [`DisplayFormatter`](https://ipython.readthedocs.io/en/stable/api/generated/IPython.core.formatters.html#IPython.core.formatters.DisplayFormatter) associated with the type, which in turn converts the instance to a display-appropriate representation, based on MIME type. For example, each `DisplayFormatter` may `plain/text` version for the IPython shell, and a `text/html` version for a Jupyter Notebook.
+
+```python imports, echo=False, results='hidden'
+from pyrasterframes.all import *
+from pyspark.sql.functions import col
+spark = create_rf_spark_session()
+```
+
+## Initialize Sample
+
+First we read in a sample image as tiles:
+
+```python raster_read
+uri = 'https://modis-pds.s3.amazonaws.com/MCD43A4.006/31/11/2017158/' \
+      'MCD43A4.A2017158.h31v11.006.2017171203421_B01.TIF'
+
+# here we flatten the projected raster structure 
+df = spark.read.raster(uri) \
+        .withColumn('tile', rf_tile('proj_raster')) \
+        .withColumn('crs', rf_crs(col('proj_raster'))) \
+        .withColumn('extent', rf_extent(col('proj_raster'))) \
+        .drop('proj_raster')
+```
+ 
+Print the schema to confirm its "shape":
+
+```python schema
+df.printSchema()
+```
+
+# Tile Display
+
+Let's look at a single tile. A `pyrasterframes.rf_types.Tile` will automatically render nicely in Jupyter or IPython.
```

Comment:
i think some of this is already covered in `raster-write.pymd`, would be nice to slim that page down and point it over here. you may also want to just lift some of that content into here.